lib/subscriber.rb

```diff
@@ -38,9 +38,6 @@ def self.valid_sleep_period?(sleep_period)
     end
   end
 
-  
-
-# Todo: URL validation /\A(https:\/\/sqs.)#{AWS["region"]}.amazonaws.com\/\d{12}\/([a-zA-Z]*\d*[-_]*)+|(.fifo)\z/
```

Comment:
We need this validation and test

Reply:
validation has already been added and is in place. So is the test. Needed to add test for invalid URL, which I did now.

Reply:
Test for queue url - for happy flow and errors